packages/docusaurus/lib/client/exports/Link.js

```diff
@@ -6,15 +6,15 @@
  */
 
 import React, {useEffect} from 'react';
-import Perimeter from 'react-perimeter';
 import {NavLink} from 'react-router-dom';
 
 const internalRegex = /^\/(?!\/)/;
 
 function Link(props) {
-  const {to, href, preloadProximity = 20} = props;
+  const {to, href} = props;
   const targetLink = to || href;
   const isInternal = internalRegex.test(targetLink);
+  let preloaded = false;
```

Comment:
This means that the `preloaded` state doesn't persist across renders. Is that intended?

Reply:
yes. It's okay anyway since the dedupe is actually happening on docusaurus.js